src/main/webapp/scripts/map.js

```diff
@@ -12,82 +12,175 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+const skyGarden = {
+  name: "Sky Garden", 
+  category: "Places to Visit", 
+  lat: 51.510881, 
+  lng: -0.083751,
+  description: "Sky Garden is a great place to visit, with really good views of London. \
+  You can enjoy the 360 degree view of the city - for free! They are public gardens and \
+  definitely worth a visit when you have the time. Remember to keep socially distanced!", 
```

Comment:
Please use an indentation for wrapping a string into multiple lines. (usually it's double the size of the block indentation so it'll be 4 spaces in this case)

Reply:
oh thanks for the tip - I wasn't sure what the indentation was meant to be but I'll be sure to remember in future now!